world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
```

Comment:
@abbacode we can write abilities as dictionary literals one after the other, like
``` python
{name: sneak, ....}

{name: leadership, ....}
````
etc

I'm looking back at the conversation here regarding __init__ but I'm not sure about your question, I think this is a reference to https://github.com/evennia/ainneve/pull/36#discussion_r35426112 ? If so you can disregard that, it's not about __init__ files but about the __init__ method on Ability (which we don't have to worry about)

To refer to abilities we would import the whole abilities.py module (i.e. file)

`from world.abilities import ....`